templates/pages/setup/general/preferences_setup.html.twig

```diff
@@ -390,11 +390,11 @@
 
    {% if call('\\Glpi\\Dashboard\\Grid::canViewOneDashboard') %}
       {{ fields.smallTitle(__('Dashboards'), call('Glpi\\Dashboard\\Dashboard::getIcon'), '', 'section-dashboards') }}
-      {% macro dropdownDashboard(name, config, context = 'core', disabled_option = false) %}
+      {% macro dropdownDashboard(name, config, thecontext = "core", disabled_option = false) %}
```

Comment:
No idea why. Maybe `context` is a kind of reserved keyword in this... context... ?